packages/api-reference/src/helpers/scrollToId.ts

```diff
@@ -2,5 +2,7 @@
  * Tiny wrapper around the scrollIntoView API
  */
 export const scrollToId = async (id: string) => {
-  document.getElementById(id)?.scrollIntoView()
+  const root =
```

Comment:
We need to check if we're in a shadow DOM in order for `getElementById` to work (there's probably a better solution for this)

Reply:
LGTM